chall04/vde-dios.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+import sys
+import re
+
+
+def error(e , i):
+    file_name =  "stdin" if i == 0 else sys.argv[i]
```

Comment:
What does `i` mean? You should use more explicit variable names. Also you are in python, it sounds here like you should use the `bool` type